chapter5.Rmd

```diff
@@ -161,7 +162,7 @@ str(mtcars)
 ```{r}
 ex() %>% check_output_expr("str(mtcars)", missing_msg = "Have you correctly called `str()` on `mtcars`?")
 
-success_msg("Nice work! Can you find all the information that is listed in the exercise's assignment? Continue to the next exercise.")
+success_msg("Nice work! You can find lots of information by viewing the `str()` of a dataset? Continue to the next exercise.")
```

Comment:
This sentence ends in a `?`, but isn't formulated as a question, I think?

Reply:
I'll hold off on merging

Reply:
Thanks for spotting @TuurDutoit, I've updated the message.